fdbserver/DataDistribution.actor.cpp

```diff
@@ -96,6 +97,8 @@ struct DDAudit {
 	bool cancelled; // use to cancel any actor beyond auditActor
 	int64_t overallIssuedDoAuditCount;
 	int64_t overallCompleteDoAuditCount;
+	AsyncVar<int> remainingBudgetForAuditTasks;
```

Comment:
Now the budget is per `Audit`, do we limit the number of audit per type?

Reply:
Yes. We allow 1 audit per type.